README.md

```diff
@@ -40,3 +40,41 @@ docker pull ghcr.io/input-output-hk/partner-chains/partner-chains-node:latest
 ## How to Use
 
 Refer to the [documentation](docs/user-guides) for detailed instructions on how to use the node and CLI tools.
+
+## Block Production Rewards for Validators
+
+The Partner Chains node provides a simple mechanism for exposing the mapping of block beneficiaries with produced blocks. 
+The Partner Chain Builder is responsible for using this input to accurately calculate and distribute block rewards on their partner chain, following the tokenomics they will design and implement. 
```

Comment:
I think in this context Partner Chain Builder is not the noun describing us, I believe it should instead be 'partner chain builder' or more simply, 'chain builder'